src/config/config-production.ts

```diff
@@ -150,12 +150,16 @@ const EXTERNAL_BASEURL = env.EXTERNAL_BASEURL || `http://localhost:${PORT}`;
 // External baseUrl for git clone urls
 const EXTERNAL_GIT_BASEURL = env.EXTERNAL_GIT_BASEURL || `http://localhost:${GITLAB_PORT}`;
 
-// URL pattern used for composing external deployment URLs
+const deploymentDomain = `deployment.localtest.me`;
```

Comment:
This is onyly used in the defaults. In practice the overriding environment variables should be specified in production.